### ISSUE TITLE
Bug: using annotations on the domain axis and a data series throws the error "Failed to execute 'replaceChild' on 'Node': The node to be replaced is not a child of this node."

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1514]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1. Create a chart with annotations on both the domain axis and data series, eg: http://jsfiddle.net/asgallant/368yU/

If the first row in the DataTable does not contain any annotation strings, then the
error is not thrown, eg: http://jsfiddle.net/asgallant/368yU/1/

What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
AreaChart, BarChart, ColumnChart, LineChart

Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
No

What operating system and browser are you using?
Chrome, Windows 7x64


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `drew_gallant@abtassoc.com` on 2014-03-04 15:15:25

**Comment by orwant** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2014-03-04 15:32:00

**Comment by orwant** [comment]
```
This is fixed in the latest candidate release (1.1) for v42.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-04-20 18:20:58
